rpcs3/Emu/Cell/lv2/sys_timer.cpp

```diff
@@ -267,7 +267,8 @@ error_code _sys_timer_start(ppu_thread& ppu, u32 timer_id, u64 base_time, u64 pe
 	{
 		std::lock_guard lock(timer.mutex);
 
-		if (!lv2_obj::check(timer.port))
+		// LV2: Simple nullptr check, nothing complex (do not use lv2_obj::check here)
```

Comment:
This comment can be improved by explaining why lv2_obj::check is incorrect here or if maybe it's just unnecessary.

Reply:
Better explained.